### DISCUSSION TITLE
The bot leaves the vc without playing a song

### DISCUSSION THREAD

**Discussion by shawn3308** [eritislami/evobot#669] [Q&A]
![image](https://user-images.githubusercontent.com/60691117/109777778-16ec7d00-7c2a-11eb-9b5a-820f49288aa7.png)
![image](https://user-images.githubusercontent.com/60691117/109777846-266bc600-7c2a-11eb-8070-030c09a6d775.png)

what can i do

**Comment by wolf-yuan-6115** [comment]
that is your internet's problem

**Reply by shawn3308** [reply]
![image](https://user-images.githubusercontent.com/60691117/109798556-e95efe00-7c40-11eb-8cb0-0b9b065b5727.png)

is this speed not good .to run the bot (●'◡'●)
